src/components/HTMLEngineProvider/HTMLRenderers/ImageRenderer.tsx

```diff
@@ -102,6 +112,7 @@ function ImageRenderer({tnode}: ImageRendererProps) {
                             shouldUseHapticsOnLongPress
                             accessibilityRole={CONST.ROLE.BUTTON}
                             accessibilityLabel={translate('accessibilityHints.viewAttachment')}
+                            disabled={!isOffline && hasLoadFailed}
```

Comment:
When we decide to disable opening images in case of a load failure, we should ensure consistency for both online and offline scenarios
```suggestion
                            disabled={hasLoadFailed}
```

Reply:
it's done